zuul-core/src/main/java/com/netflix/zuul/netty/server/SocketAddressProperty.java

```diff
@@ -157,11 +157,6 @@ public SocketAddress apply(String input) {
             }
             throw new AssertionError("Missed cased: " + bindType);
         }
-
-        @Override
-        public boolean equals(Object object) {
-            return false;
```

Comment:
😅

Reply:
Yeah not sure why this was here, errorprone complains that it doesn't implement hashCode as well but I'm not sure we need it in the first place.